### DISCUSSION TITLE
The aptos specification

### DISCUSSION THREAD

**Discussion by gftea** [aptos-labs/aptos-developer-discussions#203] [Questions] [answered]
### Describe your question in detail.

In aptos-core, the links to below documents are broken, where can I find the docs?

* To understand the authenticated blockchain data structure, see
https://github.com/aptos-labs/aptos-core/tree/main/documentation/specifications/common
* To learn more about the backup system including the backup data format, see
https://github.com/aptos-labs/aptos-core/blob/main/documentation/specifications/db_backup/spec.md

### What have you tried or looked at?  Or how can we reproduce the error?

can not find

### Which operating system are you using?

N/A

### Which SDK or tool are you using? (if any)

N/A

**Comment by 0xmaayan** [comment, marked as answer]
Here’s the last copy of the specifications https://github.com/aptos-labs/aptos-core/tree/aptos-node-v1.2.0/documentation

They may be incorrect, as they are from Libra / Diem days, with not much updates.  
These were removed, as they were presumed to be outdated.
